guides/release/components/triggering-changes-with-actions.md

```diff
@@ -103,34 +103,37 @@ in the child component we will implement the logic to confirm that the user want
 
 ```javascript {data-filename=app/components/button-with-confirmation.js}
 import Component from '@ember/component';
+import { action } from '@ember/object';
 
-export default Component.extend({
-  actions: {
-    launchConfirmDialog() {
-      this.set('confirmShown', true);
-    },
+export default class ButtonWithConfirmation extends Component {
+  @action
+  launchConfirmDialog() {
+    this.set('confirmShown', true);
```

Comment:
@locks @pzuraq There is no `this.set` in GCs, correct? Also I'd have to use `@tracked`?

Reply:
Confirm to both!

Reply:
`import Component, { tracked } from '@glimmer/component';`?

Reply:
I believe so.